api/v1alpha1/envoyproxy_tracing_types.go

```diff
@@ -34,6 +34,15 @@ type ProxyTracing struct {
 	//
 	// +optional
 	CustomTags map[string]CustomTag `json:"customTags,omitempty"`
+	// Tags defines the custom tags to add to each span.
+	// Envoy [command operators](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#command-operators) may be used in the value.
+	// The [format string documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#config-access-log-format-strings) provides more information.
+	// If provider is kubernetes, pod name and namespace are added by default.
+	//
+	// Same keys take precedence over CustomTags.
+	//
+	// +optional
+	Tags map[string]string `json:"tags,omitempty"`
```

Comment:
Will there ever be a usecase where the tag value comes from a configmap or secret in k8s instead of hardcoded literal? If so, we may want to reconsider the value type `string`, we can change it to `CustomTagValue` struct that can later support non-literal values.